bundles/org.openhab.binding.enturno/src/main/java/org/openhab/binding/enturno/internal/model/estimated/EstimatedCalls.java

```diff
@@ -0,0 +1,161 @@
+/**
+ * Copyright (c) 2010-2019 Contributors to the openHAB project
+ *
+ * See the NOTICE file(s) distributed with this work for additional
+ * information.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ */
+package org.openhab.binding.enturno.internal.model.estimated;
+
+/**
+ * Generated Plain Old Java Objects class for {@link EstimatedCalls} from JSON.
+ *
+ * @author Michal Kloc - Initial contribution
+ */
+public class EstimatedCalls
+{
+    private String aimedArrivalTime;
```

Comment:
Please use public fields for DTO (data transfer objects) and no getters/setters. If you have used a generator, please configure the generator to respect this preference.

Reply:
I don't think we have this as a standard and it's just a personal preference to use or not use getters/setters. So I think the code is just fine with the getters and setters and there is no need to change it.

Reply:
Done